src/components/hero/hero.test.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+// got issue Hero.tsx is not a module
+// what prevented success compilation
+// that's why added
+// @ts-ignore
```

Comment:
Suppressing typescript errors is a bad practice. You might need to:
1. Install `@types/testing-library__jest-dom`;
2. Append `"jest", "@testing-library/jest-dom"` to `tsconfig.json`>`"types"` and remove `"cypress"` from there (its types are conflicting with `jest-dom` types);
3. Put cypress types settings inside `/cypress/tsconfig.json`:
```json
{
    "extends": "../tsconfig.json",
    "compilerOptions": {
      "baseUrl": "../node_modules",
      "types": ["cypress"]
    },
    "include": [
      "**/*.ts"
    ]
  }
```